backend/vouch/voting_rights.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
```

Comment:
Maybe this algorithm should be moved to backend/ml/ ? It's not directly related to the vouching mechanism.

Reply:
yeah, I hesitated. I'm thinking that `ml` is quite a bad name.

Reply:
I also think `ml` is not appropriate anymore. It could be a good move to rename it one day.

Few thoughts:

`ml` is very very generic, we could have several Django app using ML algorithms in the back end, in this case they will have names based on their purpose and not based on what they are. Using `ml` for a Django app is a bit like naming a variable `integer` or `string`.